screens/MeetYourMatch.js

```diff
@@ -10,6 +11,14 @@ export class MeetMatch extends React.Component {
   state = {
     fontLoaded: false,
   };
+
+  componentWillMount() {
+    const gameId = this.props.navigation.state.params.id
+    getGame(gameId).then((game) => {
+      this.setState({game})
```

Comment:
There is not `state` variable named `game`. Either remove it or create the variable.